### PR TITLE
Remove cases and deaths from trends

### PR DESCRIPTION
**Pull request by faihegberg**
- Remove daily cases and deaths from Trends.
- Select "Weekly new cases per 100k" by default in Trends.

As per these two cards:
- https://trello.com/c/XaHrtmmV/549-trends-remove-daily-new-cases-daily-new-deaths
- https://trello.com/c/toG5398D/554-update-trends-add-weekly-new-deaths-minor-updates

### CODE REVIEW

File: src/components/Explore/interfaces.ts [hunk 1]
```diff
@@ -21,19 +21,26 @@ export interface Series {
   params?: SeriesParams;
 }
 
+/**
+ * Indices assigned to each enumerated explore metric needs to match
+ * the order of the dropdown items in EXPLORE_METRICS in Explore/utils.ts.
+ *
+ * So if we change the ordering of items in EXPLORE_METRIC,
```
Comment (chasulin) [inline src/components/Explore/interfaces.ts:28]: EXPLORE_METRICS*

id be explicit and say `EXPLORE_METRICS in Explore/utils.ts`

**Review by chasulin — APPROVED** [review]

File: src/components/Explore/interfaces.ts [hunk 2]
```diff
@@ -21,19 +21,26 @@ export interface Series {
...
-  VACCINATIONS_FIRST_DOSE,
-  VACCINATIONS_COMPLETED,
-  VACCINATIONS_ADDITIONAL_DOSE,
-  ICU_USED,
-  POSITIVITY_RATE,
-  ADMISSIONS_PER_100K,
-  RATIO_BEDS_WITH_COVID,
-  WEEKLY_CASES_PER_100K,
+  CASES = 10,
+  DEATHS = 11,
```
Comment (chasulin) [inline src/components/Explore/interfaces.ts:33]: lets move these to the bottom with a comment, like: 

```
export enum ExploreMetric {
  HOSPITALIZATIONS = 0,
  ICU_HOSPITALIZATIONS = 1,
  VACCINATIONS_FIRST_DOSE = 2,
  VACCINATIONS_COMPLETED = 3,
  VACCINATIONS_ADDITIONAL_DOSE = 4,
  ICU_USED = 5,
  POSITIVITY_RATE = 6,
  ADMISSIONS_PER_100K = 7,
  RATIO_BEDS_WITH_COVID = 8,
  WEEKLY_CASES_PER_100K = 9,

// Cases and Deaths no longer used in Trends (ie. no longer included in EXPLORE_METRICS in Explore/utils.ts), and so we order them last in the enum
 CASES = 10,
  DEATHS = 11,
}
```

**Comment by faihegberg** [comment]
[Update Trends - Add weekly new deaths + minor updates](https://trello.com/c/toG5398D/554-update-trends-add-weekly-new-deaths-minor-updates)

**Comment by faihegberg** [comment]
[Trends: Remove Daily new cases, Daily new deaths](https://trello.com/c/XaHrtmmV/549-trends-remove-daily-new-cases-daily-new-deaths)

**Pull request merged**